python/cudf/cudf/core/dtypes.py

```diff
@@ -12,6 +12,7 @@
 
 import cudf
 from cudf._typing import Dtype
+from cudf.core.buffer import Buffer
 
 
 class _BaseDtype(ExtensionDtype):
```

Comment:
Should `_BaseDtype` extend `Serializable` now?

Reply:
With the changes, you've just added it should now. I'm actually not 100% sure what we should and shouldn't do with _BaseDtype. Ashwin is probably the right person to ask about this, though IMO this should be fine 😄

Reply:
Would agree. Our dtypes should probably all be `Serializable`.